packages/webapp/src/utils/developer-constants.js

```diff
@@ -194,11 +165,11 @@ export default {
     DEVELOPER_ROLE_KEY,
     DEVELOPER_IS_AGENT,
     DEVELOPER_DEFRA_ACCOUNT_DETAILS_CONFIRMED,
-    DEVELOPER_LANDOWNER_OR_LEASEHOLDER,
+    DEVELOPER_LANDOWNER_OR_LEASEHOLDER: routes.DEVELOPER_LANDOWNER_OR_LEASEHOLDER,
     DEVELOPER_LANDOWNER_TYPE,
-    DEVELOPER_CLIENT_INDIVIDUAL_ORGANISATION,
-    DEVELOPER_CLIENTS_NAME,
-    DEVELOPER_CLIENTS_ORGANISATION_NAME,
+    DEVELOPER_CLIENT_INDIVIDUAL_ORGANISATION: routes.DEVELOPER_CLIENT_INDIVIDUAL_ORGANISATION,
+    DEVELOPER_CLIENTS_NAME: routes.DEVELOPER_CLIENTS_NAME,
+    DEVELOPER_CLIENTS_ORGANISATION_NAME: routes.DEVELOPER_CLIENTS_ORGANISATION_NAME,
```

Comment:
If you revert the changes to the routes above, won't need these redis keys defined this way

Reply:
Done